tools/restatectl/src/environment/task_center.rs

```diff
@@ -58,7 +58,7 @@ where
         .expect("task_center builds")
         .into_handle();
 
-    let result = task_center.run_sync(|| fn_body(config)).await;
+    let result = task_center.run_async(|| fn_body(config)).await;
```

Comment:
The way to run the fn_body generated future in the task center context can be expressed via

```
let result = fn_body(config).in_tc(&task_center).await;
```

Reply:
Thanks for your advise, fixed.